erts/doc/src/erlang.xml

```diff
@@ -2099,6 +2099,13 @@ true</pre>
             formatted using scientific notation with <c>Decimals</c>
             digits of precision.</p>
           </item>
+          <item><p>If option <c>short</c> is specified, the float is formatted
+            using whichever notation give the shortest amount of characters
+            that still allow a round trip conversion without any precision loss.
```

Comment:
I think this sentence is a little bit confusing. It is not the notation that decides if there is any lose in precision. Perhaps something like this is better:

If option <c>short</c> is used, the float is formatted with the smallest number of digits that still guarantees that <c>F =:= list_to_float(float_to_list(F, [short]))</c>. When the float is inside the range (-2^53, 2^53), the notation that yields the smallest number of characters is used (scientific notation or normal decimal notation). Floats outside the range (-2^53, 2^53) are always formatted using scientific notation to avoid confusing results when doing arithmetic operations.

Reply:
Like this ?

Reply:
Yes, I think it is better now.